fdbserver/GlobalTagThrottler.actor.cpp

```diff
@@ -1025,3 +1075,47 @@ TEST_CASE("/GlobalTagThrottler/ReservedQuota") {
 	wait(timeoutError(monitor || client || updater, 600.0));
 	return Void();
 }
+
+// Test that tags are expired iff a sufficient amount of time has passed since the
+// last transaction with that tag
+TEST_CASE("/GlobalTagThrottler/ExpireTags") {
```

Comment:
Is there a way to trigger the `monitorThrottlingChanges` loop so we can also test that the expired tag automatically gets removed?

Reply:
`monitorThrottlingChanges` is not run in the unit test environment because it requires a full cluster. I have another PR open to improve full end-to-end simulation testing, and will ensure that `monitorThrottlingChanges` works correctly with these end-to-end tests.